var/spack/repos/builtin/packages/cp2k/package.py

```diff
@@ -580,46 +608,20 @@ def edit(self, spec, prefix):
             if "+mpi_f08" in spec:
                 cppflags.append("-D__MPI_F08")
 
-            if "wannier90" in spec:
-                cppflags.append("-D__WANNIER90")
-                wannier = join_path(spec["wannier90"].libs.directories[0], "libwannier.a")
-                libs.append(wannier)
-
         if "+libint" in spec:
             cppflags += ["-D__LIBINT"]
 
-            if "@:6.9" in spec:
-                cppflags += ["-D__LIBINT_MAX_AM=6", "-D__LIBDERIV_MAX_AM1=5"]
-
-                # libint-1.x.y has to be linked statically to work around
-                # inconsistencies in its Fortran interface definition
-                # (short-int vs int) which otherwise causes segfaults at
-                # runtime due to wrong offsets into the shared library
-                # symbols.
-                libs.extend(
-                    [
-                        join_path(spec["libint"].libs.directories[0], "libderiv.a"),
-                        join_path(spec["libint"].libs.directories[0], "libint.a"),
-                    ]
-                )
-            else:
-                fcflags += pkgconf("--cflags", "libint2", output=str).split()
-                libs += pkgconf("--libs", "libint2", output=str).split()
+        fcflags += pkgconf("--cflags", "libint2", output=str).split()
+        libs += pkgconf("--libs", "libint2", output=str).split()
```

Comment:
Is the indentation correct here? Currently it is out of `if "+libint" in spec:`.